universal-login-react/src/core/utils/isPrefilledAmountUsedBy.ts

```diff
@@ -0,0 +1,7 @@
+import {TopUpProvider} from '../models/TopUpProvider';
+
+const providersUsedPrefilledAmount = [TopUpProvider.RAMP];
+
+export const isPrefilledAmountUsedBy = (topUpProvider?: TopUpProvider) => {
```

Comment:
Why did you move it out of the service, that seemed like a better place for this function?

Reply:
👍